.github/stale.yml

```diff
@@ -1,11 +1,11 @@
 # Configuration for probot-stale - https://github.com/probot/stale
 
 # Number of days of inactivity before an Issue or Pull Request becomes stale
-daysUntilStale: 90
+daysUntilStale: 30
```

Comment:
Far too aggressive IMO.

Reply:
TBH @klauspost 1month is really long for issues, I don't see we should keep issues beyond 30 days which are in-active.